ztp/Makefile

```diff
@@ -1,14 +1,10 @@
-POLICYGEN_DIR := ./ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator
+POLICYGEN_DIR := ./ztp-policy-generator/
 
-export GOFLAGS := $(if $(GOFLAGS),$(GOFLAGS),-mod=vendor)
+.PHONY: ci-job build-policygen
 
-.PHONY: ci-job govet gotest
-ci-job: govet gotest
+ci-job: build-policygen
 
-govet:
-	@echo "ZTP: Running 'go vet'"
-	go vet $(POLICYGEN_DIR)/...
+build-policygen:
+	@echo "ZTP: Build policy generator and run test"
+	cd $(POLICYGEN_DIR)/ && make all
```

Comment:
This is better written as `$(MAKE) -C $(POLICYGEN_DIR) all`

Reply:
done